src/main/kotlin/org.danilopianini.gradle.mavencentral/PublishOnCentral.kt

```diff
@@ -71,20 +71,20 @@ class PublishOnCentral : Plugin<Project> {
                 project.configureRepository(extension.mavenCentral)
             }
```

Comment:
I'm not 100% sure what this code is doing, but `.afterEvaluate { }` is a bit risky - it turns into a race-to-the-bottom as all the plugins try to be the last to evaluate. It's the kind of thing that can be replaced by using lazy config. `afterEvaluate { }`

Reply:
Looks good! 

I don't think I have a 'Resolve conversation' button to click? I can't see it

![image](https://user-images.githubusercontent.com/897017/173128688-47d7fc2c-c671-410c-9340-dae075234738.png)


> Are there other (better) ways to tell Gradle "run this thing only after all properties have been set"?

Using task-config avoidance :) Sometimes `afterEvaluate { }` will do the job though.